packages/contracts/contracts/Unirep.sol

```diff
@@ -52,6 +52,8 @@ contract Unirep is IUnirep, VerifySignature {
         IVerifier _epochKeyVerifier,
         IVerifier _epochKeyLiteVerifier
     ) {
+        // see IUnirep.sol EpochKeyData.data
+        require(_config.fieldCount < 128, 'datasize');
```

Comment:
I am not sure if this should be reverted by an error (to be consistent with the following codes)

but I found that in our deploy scripts, the error cannot be shown because we retry if it fails ( so it would be difficult to detect if the transaction fails because of which reason)

Reply:
it also reminds me that we should have tests for deploy scripts
we just assume it succeed in current tests

Reply:
yea, i think we don't need an error code for this. In the deploy script we should check the `config` values before trying to send the deploy transaction.